README.md

```diff
@@ -63,6 +63,17 @@ Check out the original [paper](https://vcai.mpi-inf.mpg.de/projects/DragGAN/) fo
 
 ## Running Locally
 
+### With Docker 
+
```

Comment:
Could you add a link to the docker official website ?

Reply:
done